packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2803,6 +2806,7 @@ export class ContainerRuntime
 				local,
 				savedOp,
 				runtimeBatch,
+				inboundResult.type === "fullBatch" ? inboundResult.groupedBatch : false,
```

Comment:
Might be worth adding _// name of argument_ kind of comment at the end - really hard to say what it's about.